data/World/Overworld.json

```
@@ -1723,7 +1723,7 @@
             "Bean Plant Fairy": "can_plant_bean and can_play(Song_of_Storms) and has_bottle",
             "Butterfly Fairy": "can_use(Sticks) and has_bottle",
             "Bug Shrub": "
-                (is_child or can_use(Hover_Boots) or (is_adult and logic_zora_river_lower)) and
+                (is_child or is_adult and (plant_beans or Hover_Boots or logic_zora_river_lower)) and
```

```suggestion
                (is_child or (is_adult and (plant_beans or Hover_Boots or logic_zora_river_lower))) and
```

While technically equivalent, logic code style requires parentheses around nested Boolean expressions.

---

That's actually what I wanted to do, sorry. Thanks for mentioning it.